doc/install_windows_manually.txt

```diff
@@ -51,27 +49,62 @@ Firebird is run as application by executing it with the -a switch:
 Installation with a different service name
 ------------------------------------------
 
-A new feature of Firebird 2.1 is the option to run multiple Firebird services
+A feature of Firebird 4.0 is the option to run multiple Firebird services
 simultaneously. To do this you need to make appropriate changes to the
 firebird.conf file and then run instsvc with the -name option. For example:
 
   instsvc install -auto -superserver -guardian -name secondservice
   instsvc start -name secondservice
 
-The install_super.bat and install_classic.bat will do this automatically for
+The install_service.bat will do this automatically for
 you. After modifying the firebird.conf file just pass the servicename as a
 parameter:
 
-  install_super.bat secondservice
+  install_service.bat secondservice
 
 NOTE: If you are running multiple Firebird services it is essential that you 
 do not configure the FIREBIRD environment variable at the system level.
 
+Initializing the Security Database
+----------------------------------
+The security database (security4.fdb) has no predefined users. This is intentional.
+You will need to create the user SYSDBA and set up the password for it 
+using SQL CREATE USER command syntax in embedded mode as your first step to getting
+access to databases and utilities.
+
+Initialization is performed in embedded mode using the isql utility. For an embedded connection, an authentication
+password is not required and will be ignored if you provide one. An embedded connection will work fine
+with no login credentials and "log you in" using your host credentials if you omit a user name. However, even
+though the user name is not subject to authentication, creating or modifying anything in the existing security
+database requires that the user be SYSDBA; otherwise, isql will throw a privilege error for the CREATE USER
+request.
+
+The SQL user management commands will work with any open database. Because the sample database employee.fdb 
+is present in your installation and already aliased in databases.conf, it is convenient to use
+it for the user management task.
+
+1. Stop the Firebird server. Firebird 4 caches connections to the security database aggressively. The presence
+   of server connections may prevent isql from establishing an embedded connection.
+2. In a suitable shell, start an isql interactive session, opening the employee database via its alias:
+    > isql -user sysdba employee
+3. Create the SYSDBA user:
+    SQL> create or alter user SYSDBA password 'StrongPassword';
```

Comment:
Please mention once more that instead words StrongPassword user should invent something unique, copy-paste here can play bad joke.

Reply:
Added warning before and after the SQL snippet.